### PR TITLE
Karo/helix log

### PR DESCRIPTION
**Pull request by koraa**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+## [4.3.3](https://github.com/adobe/helix-status/compare/v4.3.2...v4.3.3) (2019-09-04)
```
Comment (trieloff) [inline CHANGELOG.md]: This does not belong to the PR, this file is auto-generated.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/helix-status",
-  "version": "4.3.0",
+  "version": "4.3.3",
```
Comment (trieloff) [inline package.json]: You should not bump the version yourself, we have semantic-release for that.

File: src/bin.js [hunk 3]
```diff
@@ -50,13 +49,23 @@ try {
...
 async function getmonitors(auth, monitorid, monitorname) {
   try {
-    const response = await request.get('https://synthetics.newrelic.com/synthetics/api/v3/monitors', {
-      headers: {
-        'X-Api-Key': auth,
-      },
-      json: true,
-    });
-    const monitors = response.monitors.map(({ id, name }) => ({ id, name }));
+    let more = true;
```
Comment (trieloff) [inline src/bin.js]: Please resolve the conflicts with the base version first. The log addition itself looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (koraa) [reply CHANGELOG.md]: I think I based my changes on the wrong branch…none of those are changes contained in the two commits I actually created…

Comment (koraa) [reply CHANGELOG.md]: There, I rebased and this seems solved…

**Comment by trieloff** [comment]
@koraa merge at will

**Comment by koraa** [comment]
@trieloff ++1 :)
